src/components/plans/PlanCreate.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {
+  Create,
+  SimpleForm,
+  TextInput,
+  BooleanInput,
+  SelectInput,
+  NumberInput,
+  required,
+} from "react-admin";
+
+const PlanCreate = (props) => {
+  return (
+    <Create label="Create" title="Create a Plan" {...props}>
+      <SimpleForm redirect="show">
+        <TextInput label="Name" source="name" validate={[required()]} />
+        <BooleanInput
+          label="Logging"
+          source="logging"
+          validate={[required()]}
+        />
```

Comment:
create does not work unless you interact with this switch
since the default is off state, we can remove `required` here..
![image](https://user-images.githubusercontent.com/17856665/88460692-f1c6ad80-ce95-11ea-9d87-1a7013204364.png)